policy/centos7/rancher.te

```diff
@@ -29,3 +29,24 @@ allow rke_logreader_t syslogd_var_run_t:dir read;
 allow rke_logreader_t syslogd_var_run_t:file { getattr open read };
 allow rke_logreader_t var_log_t:dir read;
 allow rke_logreader_t var_log_t:file { getattr open read };
+
+########################
+# type rke_container_t #
+########################
+gen_require(`
+        type container_runtime_t, unconfined_service_t;
+        type container_log_t;
+        type syslogd_var_run_t;
+        type var_log_t;
```

Comment:
`syslogd_var_run_t` and `var_log_t` aren't used in the rules below so they shouldn't be needed